source/includes/_endpoint-dataset.md

```diff
@@ -692,18 +692,19 @@ Refer to the options described on the table above for the `csv` format to change
 
 ##### Match
 
-The match endpoint provides a graph of matches indicating which variables match amongst the datasets provided.  To use it,
+The match endpoint provides a list of matches indicating which variables match amongst the datasets provided.  To use it,
 send a post request representing an ordered list of datasets you would like to match. Include the "minimum_matches"
 parameter in your graph if you would like to limit the output of the matches based on the number of datasets matching.
-The default minim_matches is 2.  Currently, only alias is utilized to match the variables to one another.  To 
+The default minim_matches is 2. Currently, only alias is utilized to match the variables to one another. 
 
-The result of a match endpoint request can be one of two things.  If the same match has been completed previously, the
-api with return a 201 status code, along with the match results.  Otherwise, the endpoint will return a 202 status code,
-with a Progress result that provides status information as the match is completed.  Either request will result in the
-location header being set to the URI for staticly generated comparison result that can be accessed with the match is completed.
+The result of a match endpoint request can be one of two things. If the same match has been completed previously, the
+api with return a 201 status code and a Location header to the existing results. Otherwise, the endpoint will return 
+a 202 status code, with a Progress result that provides status information as the match is completed. Either request will 
+result in the location header being set to the URI for staticly generated comparison result that can be accessed with the 
```

Comment:
you mention the location header in two different places, perhaps you can consolidate.